jormungandr-lib/src/interfaces/block0_configuration/DOCUMENTED_EXAMPLE.yaml

```diff
@@ -118,26 +118,37 @@ blockchain_configuration:
   # Halving formula: constant * ratio ^ (#epoch after epoch_start / epoch_rate)
   #
   reward_parameters:
-    halving: # or use "linear" for the linear formula
+    linear: # or use "linear" for the linear formula
```

Comment:
I don't understand why you needed to change this value?

Reply:
We will be using a constant return for the testnet

Reply:
this is a `DOCUMENTED_EXAMPLE.md` not the settings for the testnet.

Reply:
That's why it's good to have the same settings where possible (avoids confusion).  You can always add another example for an alternative setup if you like.

Reply:
I have created an alternative setup. It's sensible to show the actual testnet parameters in a config file

Reply:
devops or comm will do that. No need to show the actual parameters here.